docs/en/core-services/Caching.md

```diff
@@ -237,7 +283,7 @@ available options are:
 #### Evictor Emulation
 The old eviction policies are now removed and Alluxio provided implementations are replaced with appropriate annotation policies.
 Configuring old Alluxio evictors will cause worker startup failure with `java.lang.ClassNotFoundException`.
-Also, the old watermark based configuration is invalidated. So the following previous configuration options are ineffective:
+Also, **the old watermark based configuration is invalidated**. So the following previous configuration options are ineffective:
```

Comment:
This kind of change is quite important actually. I don't think we have a notice to those who are relying on this watermark feature.